### ISSUE TITLE
Train Loss Test Loss

### ISSUE THREAD

**Issue by JoltonDsouza** [YunYang1994/tensorflow-yolov3#248]
For the first several epochs, the test loss goes down as the training loss goes down but after a while, the test loss remains constant and then goes up. What's the matter?

**Comment by MichaelCong** [comment]
过拟合
